tests/test_ssh.py

```diff
@@ -0,0 +1,35 @@
+import time
+import pexpect
+import sys
+
+def configure_network(dut):
+    # Configures the LAN network to DHCP client mode
+    # OpenWrt will get an IP address from the qemu DHCP server
+    dut.send_cmd("ip addr", "192.168.1.1/24")
+    dut.send_cmd("uci set network.lan.proto=dhcp")
+    dut.send_cmd("uci changes", "network.lan.proto='dhcp'")
+    dut.send_cmd("uci commit")
+    dut.send_cmd("service network reload", "entered forwarding state")
+    # Wait for DHCP configuration
+    time.sleep(8)
+
+    dut.send_cmd("ip addr", "10.0.2.15/24")
+
+def test_opkg(dut):
+    configure_network(dut)
```

Comment:
Wouldn't it be easier to log in via link local and give the LAN interface a specified MAC address? The network setup stuff should be it's own test

Reply:
This interface is not bridged to the host. The qemu command arguments we use will forward packets we send to port 8022 to port 22 of this qemu instance. This is only  virtual network we can not really access from the host.